src/libical/icalrecur.c

```diff
@@ -991,7 +991,7 @@ int icalrecur_iterator_sizeof_byarray(short *byarray)
 
 enum expand_table
 {
-    UNKNOWN = 0,
+    IGNORE = 0,
```

Comment:
`UNKNOWN` and `IGNORE` have quite different meanings, even if the technical effect might be the same. Maybe a distinct value would make sense.

Reply:
Yes, I did exactly that in PR #429.  I had forgotten that I wrote that when I did this one, which I'm going to close.  Please address further comments on that PR